Sources/SwiftSyntax/Syntax.swift

```diff
@@ -365,20 +365,6 @@ extension Syntax: Identifiable {
 }
 
 extension Syntax {
-  /// Enumerate all of the syntax text present in this node, and all
-  /// of its children, to give a source-accurate view of the bytes.
-  ///
-  /// Unlike `description`, this provides a source-accurate representation
-  /// even in the presence of malformed UTF-8 in the input source.
-  ///
-  /// The ``SyntaxText`` arguments passed to the visitor are only guaranteed
-  /// to be valid within that call. It is unsafe to escape the `SyntaxValue`
-  /// values outside of the closure.
-  @_spi(RawSyntax)
-  public func withEachSyntaxText(body: (SyntaxText) throws -> Void) rethrows {
-    try raw.withEachSyntaxText(body: body)
```

Comment:
This function was not used. It's not worth to expose this in `Syntax` it's a SPI anyway.